spec/models/topic_spec.rb

```diff
@@ -909,7 +909,15 @@ def expect_the_right_notification_to_be_created(inviter, invitee)
   context 'private message' do
     let(:coding_horror) { Fabricate(:coding_horror) }
     fab!(:evil_trout) { Fabricate(:evil_trout) }
-    let(:topic) { Fabricate(:private_message_topic, recipient: coding_horror) }
+    let(:topic) do
+      PostCreator.new(
```

Comment:
Just for context, what's the difference between this and the previous code (`Fabricate`)? 🙂

Reply:
When `PostCreator` creates a new topic it loads the `allowed_groups` of the topic. I haven't checked where, but it might be in `PostCreator#ensure_in_allowed_users`. `Fabricate` doesn't do that and that's why the existing spec worked even though it should have failed, because `PostAlerter#notify_group_summary` didn't create a notification for a non-fabricated topic.